ros2topic/ros2topic/verb/echo.py

```diff
@@ -99,22 +105,34 @@ def main(args):
         if message_type is None:
             raise RuntimeError('Could not determine the type for the passed topic')
 
+        future = None
+        if args.once:
+            future = Future()
+            callback = subscriber_cb_once_decorator(callback, future)
```

Comment:
@craigh92 meta: no need to change this PR, but it'd be almost trivial to generalize this to echo N messages (maybe a `---times N` argument?). Then `--once` becomes syntactic sugar.